Formula/ipmitool.rb

```diff
@@ -16,6 +16,7 @@ class Ipmitool < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on arch: :x86_64
```

Comment:
It has an `arm64_big_sur` bottle, so it's probably a different issue

Reply:
I think this is weird case, the tool is not supposed to support x86 arch.

Reply:
We might need a new rebuild to double confirm if it is still buildable for `arm64_big_sur`

Reply:
The issue was a change made in newer Xcode that only affected arm64.

The fix is probably to stop the `#pragma pack(1)` bleeding everywhere (i.e. make it `pack(push,1)` and then `pack(pop)` at the end of the header).

Reply:
> The fix is swap the order of vFlashstr, if possible. With the #pragma pack(1), the pointer is starting on byte 2.

Cool, I can maybe patch this in the upstream.